### PR TITLE
Add debugger config

### PR DESCRIPTION
**Pull request by Krzysztof-Cieslak**
596760f812e1b998067789

### CODE REVIEW

**Comment by Krzysztof-Cieslak** [comment]
I've added text in readme ;P

**Comment by forki** [comment]
💋

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+            "cwd": "${workspaceRoot}",
+            "stopAtEntry": false,
+            "console": "internalConsole"
+        },
+        {
+            "name": "Debug Server Test",
+            "type": "coreclr",
+            "request": "launch",
+            "preLaunchTask": "Build ServerTests",
+            "program": "${workspaceRoot}/test/ServerTests/bin/Debug/netcoreapp1.1/ServerTests.dll",
```
Comment (forki) [inline .vscode/launch.json]: somehow we should get rid of this here!?

Comment (Krzysztof-Cieslak) [inline .vscode/launch.json]: Not really sure how, that's just how Code debugger configuration works

Comment (Krzysztof-Cieslak) [inline .vscode/launch.json]: How it should look like? Maybe we can add some hack around it

Comment (forki) [inline .vscode/launch.json]: yeah I have no idea. it's a bit weird ;-(

**Pull request merged**